heuristics/newlines.go

```diff
@@ -97,6 +97,17 @@ func RecutNewLines(lines []string) []string {
 			continue
 		}
 
+		// If the line starts with ref: its a reference and should be on its
+		// own line
```

Comment:
nitpick (non-blocking): spelling

```suggestion
		// If the line starts with `ref:` it's a reference and should be on its
		// own line
```